ffi/diplomat/tests/missing_apis.txt

```diff
@@ -123,3 +123,4 @@ icu::relativetime::provider::ShortWeekRelativeTimeFormatDataV1Marker#Struct
 icu::relativetime::provider::ShortYearRelativeTimeFormatDataV1Marker#Struct
 icu::relativetime::provider::SingularSubPattern#Struct
 icu::relativetime::provider::SingularSubPattern::try_from_str#FnInStruct
+icu::segmenter::RuleStatusType#Enum
```

Comment:
suggestion: we probably want to expose `is_word_like()` and `rule_status` in FFI (e.g. in `segmenter_word.rs`). They can be added in a separate PR if you like.

Reply:
After fixing this, I will create new PR